packages/react-dom/src/events/__tests__/SyntheticEvent-test.js

```diff
@@ -46,7 +46,7 @@ describe('SyntheticEvent', () => {
 
     document.body.appendChild(container);
 
-    var event = createEvent('click', {srcElement: instance});
```

Comment:
I didn't ask to remove it from *all* tests. I was saying that only one test intentionally tested for `srcElement` support (that's why the test existed). That's why after refactoring, there should *also * be one test that verifies this.